sympy/functions/elementary/hyperbolic.py

```diff
@@ -1409,6 +1436,31 @@ def _eval_as_leading_term(self, x, logx=None, cdir=0):  # acosh
                 return self.rewrite(log)._eval_as_leading_term(x, logx=logx, cdir=cdir)
         return self.func(x0)
 
+    def _eval_nseries(self, x, n, logx, cdir=0):  # acosh
+        arg = self.args[0]
+        arg0 = arg.subs(x, 0)
+
+        # Handling branch points
+        if arg0 in (S.One, S.NegativeOne):
+            res = acos(arg)*sqrt(arg - 1)/sqrt(1 - arg)
```

Comment:
This looks inefficient: computing two power series and dividing them to get `I` or `-I`. There must be a way to determine the multiplier before calling `_eval_nseries`.